frontend/lib/verification/scanner/verification_encoder.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:convert';
+
+import 'package:fixnum/fixnum.dart';
+
+import '../../identification/base_card_details.dart';
+import '../../identification/protobuf/card_verify_model.pb.dart';
+import '../verification_card_details.dart';
+
+String encodeVerificationCardDetails(
+    VerificationCardDetails verificationCardDetails) {
+  final cardDetails = verificationCardDetails.cardDetails;
+
+  final cardVerifyModel = CardVerifyModel();
+
+  cardVerifyModel.fullName = cardDetails.fullName;
+  cardVerifyModel.hashSecret =
+      Base64Decoder().convert(cardDetails.hashSecretBase64);
+  cardVerifyModel.expirationDate = Int64(cardDetails.unixExpirationDate);
+  switch (cardDetails.cardType) {
+    case "STANDARD":
+      cardVerifyModel.cardType = CardVerifyModel_CardType.STANDARD;
+      break;
+    case "GOLD":
+      cardVerifyModel.cardType = CardVerifyModel_CardType.GOLD;
+      break;
+    default:
+      throw Exception(
+          "The provided card type ${cardDetails.cardType} is unknown!");
+  }
+  cardVerifyModel.regionId = cardDetails.regionId;
+  cardVerifyModel.otp = verificationCardDetails.otp;
```

Comment:
Why do we assign these variables manually? Does `CardVerifiyModel` not have a constructor which assigns the properties itself? That would seem less error-prone to me..